packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -80,79 +84,93 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
 
     const list_with_category = () => {
-        const contract_type_category_icon = getContractTypeCategoryIcons();
+        const contract_type_category_icon: { [key: string]: string } = getContractTypeCategoryIcons();
         const order_arr = ['Accumulators', 'Multipliers', 'Vanillas', 'Ups & Downs', 'Highs & Lows', 'Digits'];
-        const ordered_list = list.sort((a, b) => order_arr.indexOf(a.key) - order_arr.indexOf(b.key));
-        const accumulators_category = ordered_list.filter(({ label }) => label === localize('Accumulators'));
-        const multipliers_category = ordered_list.filter(({ label }) => label === localize('Multipliers'));
-        const options_category = ordered_list.filter(
+        list?.sort((a, b) => order_arr.indexOf(a.key) - order_arr.indexOf(b.key));
+        const accumulators_category = list?.filter(({ label }) => label === localize('Accumulators'));
+        const multipliers_category = list?.filter(({ label }) => label === localize('Multipliers'));
+        const options_category = list?.filter(
             ({ label }) => label !== localize('Multipliers') && label !== localize('Accumulators')
         );
 
-        const categories = [];
+        const categories: TList[] = [];
 
-        if (list.length > 0) {
+        if (list && list.length > 0) {
             categories.push({
                 label: localize('All'),
-                contract_categories: [...ordered_list],
+                contract_categories: [...list],
                 key: 'All',
+                icon: '',
             });
         }
 
-        if (multipliers_category.length > 0) {
+        if (multipliers_category && multipliers_category.length > 0) {
             categories.push({
                 label: localize('Multipliers'),
                 contract_categories: multipliers_category,
                 key: 'Multipliers',
+                icon: '',
             });
         }
 
-        if (options_category.length > 0) {
+        if (options_category && options_category.length > 0) {
             categories.push({
                 label: localize('Options'),
                 contract_categories: options_category,
+                // @ts-expect-error The type of the component prop in VerticalTab.Headers is wrong and it should be JSX.Element
                 component: options_category.some(category => category.key === 'Vanillas') && (
                     <span className='dc-vertical-tab__header--new'>{localize('NEW')}!</span>
                 ),
                 key: 'Options',
             });
         }
 
-        if (accumulators_category.length > 0) {
+        if (accumulators_category && accumulators_category.length > 0) {
             categories.push({
                 label: localize('Accumulators'),
                 contract_categories: accumulators_category,
+                // @ts-expect-error The type of the component prop in VerticalTab.Headers is wrong and it should be JSX.Element
```

Comment:
@henry-deriv Please update `component` prop type in `packages/components/src/components/vertical-tab/vertical-tab-header.tsx` accordingly instead of this comment. It's necessary. Please also kindly remember to check if no TS errors appear after your update in other places where VerticalTabHeader is used.

Reply:
done